app/lib/forms/choose_private_childcare_provider.rb

```diff
@@ -4,9 +4,11 @@ class ChoosePrivateChildcareProvider < Base
 
     attr_accessor :institution_name, :institution_identifier
 
-    validates :institution_identifier, format: { with: /\APrivateChildcareProvider-\d+\z/, unless: -> { institution_identifier.blank? || institution_identifier == "other" } }
+    validates :institution_identifier, format: { with: /\APrivateChildcareProvider-.+\z/, unless: -> { institution_identifier.blank? || institution_identifier == "other" } }
```

Comment:
Ofsted URNs are often prefixed with EY. I've left the formatting open ended and instead validated the institution presence later on.